sdk/metricsadvisor/azure-ai-metricsadvisor/tests/test_metrics_advisor_client_live.py

```diff
@@ -100,68 +127,95 @@ def test_list_metrics_series_data(self):
         ))
         assert len(results) > 0
 
-    def test_list_metric_series_definitions(self):
-        results = list(self.client.list_metric_series_definitions(
+    @pytest.mark.parametrize("credential", CREDENTIALS, ids=ids)
+    @MetricsAdvisorPreparer()
+    @recorded_by_proxy
+    def test_list_metric_series_definitions(self, client):
+        results = list(client.list_metric_series_definitions(
             metric_id=self.metric_id,
             active_since=datetime.datetime(2021, 1, 1),
         ))
         assert len(results) > 0
 
-    def test_add_anomaly_feedback(self):
+    @pytest.mark.parametrize("credential", API_KEY, ids=ids)  # only using API key for now since service issue with AAD
```

Comment:
Could you tell me why here we only run upon api_key?

Reply:
Was seeing BadRequest returned from the service like what karishma reported [here](https://teams.microsoft.com/l/message/19:11b7faed215e4d22bfe8b60d4a716474@thread.skype/1636749642057?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1636749642057&teamName=Azure%20SDK&channelName=Service%20-%20Metrics%20Advisor&createdTime=1636749642057)